sirepo/reply.py

```diff
@@ -610,9 +610,9 @@ def _gen_exception_reply_WWWAuthenticate(self, args):
             401, headers=PKDict({"WWW-Authenticate": 'Basic realm="*"'})
         )
 
-    def _gen_http_exception(self, code, headers=None):
+    def _gen_http_exception(self, status, headers=None):
         return self.from_kwargs(
-            content=_HTTPException(PKDict(code=code, headers=headers))
+            content=_HTTPException(PKDict(status=status, code=status, headers=headers))
```

Comment:
I think setting status here is at the wrong level of abstraction. In the http_response case we set the status in _http_error which is at a higher level of abstraction where we know about how code should be converted to status. In the websocket case http status doesn't even really make sense. There are no http status codes when you're sending websocket messages. So, this needs to be moved somewhere else or eliminated. 

To help me understand what path to take can you explain why we had to add status here? I remember doing it previously but now can't find the case where it is needed...

Reply:
I just changed that code back. It was a relic of a suggestion we discussed when we were trying to tease apart the problem.

From our slack convo:
" Its confusing who is "wrong" (server vs gui). Maybe we should set status in the server? Or maybe we should read code in the gui? Not sure. The server code is rather tangled with all of the different options of code paths so it is pretty hard to tell. I changed sirepo.reply to look like
```
    def _gen_http_exception(self, status, headers=None):
        return self.from_kwargs(
            content=_HTTPException(PKDict(status=status, code=status, headers=headers))
        )
```
Now it has both status and code in it."

The issue actually seems like it was on the client side. msgRouter in sirepo.js needed to change how it handles httpException. 
Before:
```python
if (n === "httpException") {
    r.status = content.status;
}
```
after:
```python
if (n === "httpException") {
     r.status = content.params.code;
}
```

Reply:
Ah, thanks.